pkgs/servers/documize-community/default.nix

```diff
@@ -4,25 +4,30 @@ buildGoModule rec {
   pname = "documize-community";
   version = "3.7.0";
 
+  patches = [ ./vendor.patch ];
```

Comment:
Why was this necessary?

Reply:
Their vendor folder was apparently hand modified + thus failed to build. Modifying like this made it build.